index.html

```diff
@@ -108,11 +108,11 @@ <h3>CUISINE</h3>
 						</div>
 						<div id="slide3" class="slide slide3">
 							<div class="box-container">
-								<div class="section1"></div>
+								<div class="section1" style="background-image: url('images/culturalpractices.jpeg'); background-size:cover;"></div>
 								<div class="section2">
 									<h3>CULTURAL PRACTICES</h3>
-									<a href="#slide4"><button class="prev-arrow"></button></a>
-									<a href="#slide2"><button class="next-arrow"></button></a>
+									<a href="#slide4"><button class="prev-arrow">&#9654;</button></a>

```

Comment:
here i added the entity for next and prev arrows for my horizontal scrolling slides